organizers/2025-06-12.html

```diff
@@ -0,0 +1,185 @@
+
+<section>
+    <img src="/img/django-london.svg" style="width: 40%; background: none" alt="Django London" class="plain" />
+    <h2>June 12th, 2025</h2>
+    <h2>Talks starting at 19:15!</h2>
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of Django London is to provide a fun, welcoming and
+        professional environment for a diverse group of people.
+    </p>
+    <p>
+        <a href="https://www.djangolondon.com/code-of-conduct/"
+            >djangolondon.com/code-of-conduct</a
+        >
+    </p>
+</section>
+
+<section>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Opus Recruitment Solutions</strong></li>
+        <li><strong>Kraken Tech</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section>
+    <img
+        data-src="/img/2022_KrakenV2_stacked_White.png"
+        style="width: 60%; border-radius: 10px"
+        alt="Kraken Tech"
+        class="plain"
+    />
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25
+            million households worldwide.
+        </em>
+    </p>
+    <p>
+        <a href="https://kraken.tech/jobs">kraken.tech/jobs</a>
+    </p>
+</section>
+
+<section>
+    <img
+        data-src="/img/jetbrains.svg"
+        style="width: 60%; background: none;"
+        alt="Jetbrains"
+        class="plain"
+    />
+    <p>
+        <em>
+            Pro tools created with {developers}, teams, and businesses in mind.
+        </em>
+    </p>
+    <p>(They make PyCharm.)</p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News</h2>
+    <h2>👩🏻‍💻 Talks</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h2>🗞️ News</h2>
+    <p>From <strong>Django News</strong>, a weekly newsletter on all things Django.</p>
+    <p><a href="https://django-news.com">django-news.com</a></p>
+</section>
+
+<section>
+    <h2>📼 DjangoCon Europe videos</h2>
+    <p>Many great talks watchable now!</p>
+    <p>
+        <a href="https://www.youtube.com/@DjangoConEurope">youtube.com/@DjangoConEurope</a>
+    </p>
+</section>
+
+<section>
+    <h2>🤖 Python 3.14 subinterpreters</h2>
+    <p>✨ <em>Feature freeze special exception</em> ✨</p>
+    <p>
+        Python 3.14 will gain <code>concurrent.intepreters</code> to run multiple Python interpreters in the same process.
+    </p>
+    <p>
+        Lower overhead than multiprocessing, better isolation and concurrency compared to threads.
+    </p>
+    <pre><code class="language-python">from concurrent import interpreters
+interp = interpreters.create()
+interp.call_in_thread(func)</code></pre>
+    <p>
+        <a href="https://peps.python.org/pep-0734/">PEP 734</a>
+    </p>
+</section>
```

Comment:
See:

1. the PEP: https://peps.python.org/pep-0734/

2. the steering council acceptance message: https://discuss.python.org/t/pep-734-multiple-interpreters-in-the-stdlib/41147/36